### ISSUE TITLE
Task error: Error: e is not iterable

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#3365]
I'm also getting this error now in output console after running a simple shell type build task

I get no errors if I disable cpptools extension

``` 
There are task errors. See the output for details.

Error: e is not iterable
Error: Cannot read property 'Symbol(Symbol.iterator)' of undefined
Error: Cannot read property 'Symbol(Symbol.iterator)' of undefined
Error: Cannot read property 'Symbol(Symbol.iterator)' of undefined
Error: Cannot read property 'Symbol(Symbol.iterator)' of undefined
```

```
{
    "label": "Debug-Build",
    "type": "shell",
    "command": "${workspaceFolder}/build.bash",
    "args": [
        "-d"
    ],
    "group": {
	"kind": "build",
        "isDefault": true
    },
    "problemMatcher": []
},
```

- OS and Version: macOS 10.14.3
- VS Code Version: 1.32.3
- C/C++ Extension Version: 0.22.1

_Originally posted by @ahmedm in https://github.com/Microsoft/vscode-cpptools/issues/3322#issuecomment-476636944_

**Comment by sean-mcmanus** [comment]
@ahmedm Hmm, I'm not able to repro this. Are you able to debug into the error to see who is generating those errors and/or what file?

If you use https://github.com/Microsoft/vscode-cpptools/tree/release you should be able to debug our TypeScript via Launch Extension.

**Comment by ghost** [comment]
From (Extension Host) Log I can see these errors are triggered by this event

`{"startup":false,"activationEvent":"onCommand:workbench.action.tasks.runTask"}`

I tried to debug via "Launch Extension" but I'm not getting anywhere with that
Versions 0.22.0 and 0.22.1 are affected 0.21.0 is not

**Comment by sean-mcmanus** [comment]
I saw the these errors a couple times (soon after I installed the C/C++ extension and then was doing TypeScript debugging), but I couldn't see any useful diagnostics and then it stopped reproing. The errors are in the Tasks panel. Seems like a VS Code bug to me, and I don't understand yet how our changes could cause this.

**Comment by ghost** [comment]
This is somehow got fixed in VS Code 1.33.0 update

**Comment by sean-mcmanus** [comment]
Okay, that's good to know.
